py-polars/polars/internals/lazy_frame.py

```diff
@@ -214,7 +215,9 @@ def inspect(self, fmt: str = "{}") -> "LazyFrame":
         ...     )
         ...     .inspect()  # print the node before the filter
         ...     .filter(pl.col("bar") == pl.col("foo"))
-        ... )
+        ... )  # doctest: +ELLIPSIS
```

Comment:
Will this be visible in the documentation? What does this do?

Reply:
No, see the screenshots in https://github.com/pola-rs/polars/pull/1950#issuecomment-985270232